### DISCUSSION TITLE
Allow opening navigation elements in new tab/window

### DISCUSSION THREAD

**Discussion by RobinB-Gbanga** [directus/directus#6774] [Feature Requests]
Thanks for developing this great tool!

I frequently want to open some view in a new tab, for example for keeping an item/record open as reference, or changing a table's layout/displays/interfaces to see the changes reflected side-by-side.

Directus doesn't allow to open any navigation element in a new tab, neither by middle-mouse-clicking it (nothing happens), nor through the right-click context menu.

Would it be possible to support this in the future? Adding this to the two topmost navigation levels ("modules" bar and "collections-navigation" bar) would already help a great deal.

Thanks for considering!

**Comment by rijkvanzanten** [comment]
Good question! We're just using "regular" vue-router links, so we'll have to do some research on the vue-router side why those aren't cmd-clickable 🤔

**Reply by nickrum** [reply]
That's probably caused by most of our `router-link`s not rendering `href` attributes. Our custom link components (like `v-button` and `v-list-item`) set the href attribute to `null` which overwrites `router-link` itself setting the attribute.
I tried to fix this when migration to Vue 3 by setting href to `undefined`, but I couldn't get it to work.

**Reply by rijkvanzanten** [reply]
Ahh good one

**Comment by nickrum** [comment]
This has since been addressed 👍

**Comment by NicksonYap** [comment]
Hi @nickrum, I'd like to ask in which version has it been fixed/addressed

because in Directus v10.4.2 

unable to open in new tab
![image](https://github.com/directus/directus/assets/5380731/3b232467-dba6-4bfd-a0e6-10c7c09dcdc6)
